tensorflow/compiler/xla/service/shape_inference.cc

```diff
@@ -2443,7 +2447,8 @@ ShapeInference::InferDegenerateDimensionBroadcastShape(HloOpcode operation,
                          ShapeUtil::HumanString(operand_shape),
                          StrJoin(slice_sizes, ", "));
 
-    if (operand_shape.rank() != number_of_indices) {
+    const int64 number_of_indices_int = number_of_indices;
```

Comment:
We should not create a local that just copies from another one, both being primitive types. It's useless.